### PR TITLE
New package: gum-0.10.0

### PR DESCRIPTION
**Pull request by zen0bit**
Reborn of #38427

Since is closed, new PR.

#### Testing the changes
- I tested the changes in this PR: **yes**

#### Local build testing
- I built this PR locally for my native architecture, (x64 glibc)

### CODE REVIEW

File: srcpkgs/gum/template [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'gum'
+pkgname=gum
+version=0.9.0
+revision=1
+build_style=go
+build_helper=qemu
+go_import_path="github.com/charmbracelet/gum"
+go_ldflags="-X main.Version=v${version}"
+short_desc="Tool for glamorous shell scripts"
+maintainer="akhil <akhil.j@outlook.in>"
```
Comment (tranzystorek-io) [inline srcpkgs/gum/template:10]: That's from the old PR

File: srcpkgs/gum/template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+	_cli=${DESTDIR}/usr/bin/gum
+	for shell in bash fish zsh; do
+		vtargetrun $_cli completion $shell > ${pkgname}.${shell}
+		vcompletion ${pkgname}.${shell} $shell
+	done
+
+	vtargetrun $_cli man > ${pkgname}.1
+	vman ${pkgname}.1
+}
```
Comment (tranzystorek-io) [inline srcpkgs/gum/template:28]: We don't use `pkgname` too much in the templates

File: srcpkgs/gum/template [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+build_style=go
+build_helper=qemu
+go_import_path="github.com/charmbracelet/gum"
+go_ldflags="-X main.Version=v${version}"
+short_desc="Tool for glamorous shell scripts"
+maintainer="akhil <akhil.j@outlook.in>"
+license="MIT"
+homepage="https://github.com/charmbracelet/gum"
+changelog="https://github.com/charmbracelet/gum/releases"
+distfiles="https://${go_import_path}/archive/v${version}.tar.gz"
```
Comment (tranzystorek-io) [inline srcpkgs/gum/template:14]:
```suggestion
distfiles="https://${go_import_path}/archive/refs/tags/v${version}.tar.gz"
```

Comment (tranzystorek-io) [reply srcpkgs/gum/template:14]: also don't use `go_import_path` here, rather the full URL

File: srcpkgs/gum/template (hunk 1 above)
Comment (zen0bit) [reply srcpkgs/gum/template]: adopted..

**Review by tranzystorek-io — APPROVED** [review]
LGTM

**Comment by zen0bit** [comment]
ups  
  
no changes

File: srcpkgs/gum/template [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'gum'
+pkgname=gum
+version=0.9.0
```
Comment (paper42) [inline srcpkgs/gum/template:3]: 0.10.0 is out

File: srcpkgs/gum/template [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+post_install() {
+	vlicense LICENSE
+
+	_cli=${DESTDIR}/usr/bin/gum
+	for shell in bash fish zsh; do
+		vtargetrun $_cli completion $shell > gum.${shell}
+		vcompletion gum.${shell} $shell
+	done
+
+	vtargetrun $_cli man > gum.1
```
Comment (paper42) [inline srcpkgs/gum/template:26]: I don't think there is a reason for this variable, it only makes the code less readable in my opinion.
```suggestion
	for shell in bash fish zsh; do
		vtargetrun ${DESTDIR}/usr/bin/gum completion $shell > gum.${shell}
		vcompletion gum.${shell} $shell
	done

	vtargetrun ${DESTDIR}/usr/bin/gum man > gum.1
```

File: srcpkgs/gum/template [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'gum'
+pkgname=gum
+version=0.10.0
+revision=1
+build_style=go
+build_helper=qemu
+go_import_path="github.com/charmbracelet/gum"
+go_ldflags="-X main.Version=v${version}"
```
Comment (paper42) [inline srcpkgs/gum/template:8]: I think we can drop the v, or is there a reason for it?

**Pull request merged**